### DISCUSSION TITLE
How can I use pandoc after "pip install"-ing it?

### DISCUSSION THREAD

**Discussion by Zixu-Toby-He** [jgm/pandoc#9947] [Q&A]
I have already had a python interperater on my Windows 11 computer, and several sites told me to `pip install pandoc` to obtain a copy, so as I did. But after that no site tells me how I can use it properly under python environment. They only gives `pandoc xxx -o xxx` something like this. 
The problem is that pandoc is a python package rather than a program in my computer. How can I use it to convert my "thesis.tex" into "thesis.docx" using the package on python? Or should I get a software version, too?

**Comment by Zixu-Toby-He** [comment]
Moreover, my thesis.tex has several "\include{part-file-number.tex}" and "\input{part-file-number.tex}"; does pandoc support this?

**Reply by jgm** [reply]
Yes, pandoc's LaTeX reader supports `\include`.  I can't comment on `pip install pandoc`; I don't know what that would do, but someone else may be able to help.

**Comment by alerque** [comment]
**Edit:** As @wlupton [points out](https://github.com/jgm/pandoc/discussions/9947#discussioncomment-9956711) the package you installed is just a library and does not install a wrapper binary. The tip I gave only applies to packages that include the binary and a wrapper to call it. There may or may not be such a package for Pandoc.

----

I'm probably now have a copy of `pandoc` but it is buried in your Python site-packages directory. You could probably locate it and make it more accessible with something like this (`bash`):

```
local pypath="$(python -c "import site; print(site.getsitepackages()[0]+'/bin')")"
PATH="$PATH:$pypath"
```

**Reply by Zixu-Toby-He** [reply]
The code you show is just adding python to PATH, which I did when I installed python.

**Comment by wlupton** [comment]
<https://pypi.org/project/pandoc> says it's just a python library and that you need to install pandoc separately.

**Reply by Zixu-Toby-He** [reply]
Oh thank you very much. It helps a lot.
